src/Runner/PhpInterpreter.php

```diff
@@ -24,7 +29,7 @@ function getVersion();
 	/**
 	 * @return bool
 	 */
-	function hasXdebug();
+	function canCodeCoverage();
```

Comment:
Hehe, I was expecting this discussion :o) The `canCodeCoverage` is nonsence and I didn't get better idea. The `canMeasureCodeCoverage` seems good.